app/containers/HomePage/HomePage.react.js

```diff
@@ -37,30 +41,38 @@ class HomePage extends React.Component {
         </h2>
         <label className={styles.label}>Change to your project name:
           <input className={styles.input} type="text"
-            onChange={this.onChangeProjectName}
+            onChange={ this.onChangeProjectName }
             defaultValue="React.js Boilerplate" value={projectName}
           />
         </label>
         <label className={styles.label}>Change to your name:
           <input className={styles.input} type="text"
-            onChange={this.onChangeOwnerName}
+            onChange={ this.onChangeOwnerName }
             defaultValue="mxstbr" value={ownerName}
           />
         </label>
-        <Button route="/readme">Setup</Button>
+        <Button handleRoute= { () => this.onChangeRoute('/readme') } >Setup</Button>
```

Comment:
I need to pass down `handleRoute` **function**, but the lint  cause error when using arrow function. Is there any method to solve this?